osbs/api.py

```diff
@@ -274,8 +274,15 @@ def _get_existing_build_config(self, build_config):
         - metadata.labels.git-repo-name AND metadata.labels.git-branch are equal
         """
 
-        git_labels = [(key, build_config['metadata']['labels'][key])
-                      for key in self._GIT_LABEL_KEYS]
+        git_labels = {
+            'repo_labels': [(key, build_config['metadata']['labels'][key])
+                            for key in self._GIT_LABEL_KEYS]
+        }
+        try:
+            git_labels['git-dir-name'] = build_config['metadata']['labels']['git-dir-name']
```

Comment:
Are we happy with 'git-dir-name'? Not, say, 'git-namespace'? (I have no preference)

Also, do we want to put it in osbs/constants.py instead of having multiple string literals in the code for it?